### PR TITLE
Glm/tde

### PR DESCRIPTION
**Pull request by glehmannmiotto**

### CODE REVIEW

File: include/detdataformats/tde/TDE16Frame.hpp [hunk 1]
```diff
@@ -22,7 +22,9 @@ namespace dunedaq {
 namespace detdataformats {
 namespace tde {
 
+static constexpr int ticks_between_adc_samples = 32;
 static constexpr int tot_adc16_samples = 4474;
+static constexpr int n_channels_per_frame = 64;
```
Comment (alessandrothea) [inline include/detdataformats/tde/TDE16Frame.hpp]: Isn't there only 1 channel per frame?

**Comment by alessandrothea** [comment]
Added 1 minor comment to address.
Also, the plural in the `get`/`set_adc_samples` methods. Shouldn't they be singular? They get/set 1 adc value at a time, right?

File: include/detdataformats/tde/TDE16Frame.hpp (hunk 1 above)
Comment (glehmannmiotto) [reply include/detdataformats/tde/TDE16Frame.hpp]: changed variable name to n_channels_per_amc

**Comment by glehmannmiotto** [comment]
signature to set/get adc fixed

**Review by roland-sipos — APPROVED** [review]
LGTM

**Review by alessandrothea — APPROVED** [review]

**Pull request merged**